src/components/Chart.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+
+import { Card, Typography } from "antd";
+import "../styles/Chart.css";
+
+import emptyCheckbox from "../assets/images/emptycheckbox.png";
+import checkbox from "../assets/images/checkbox.png";
+
+interface ChartProps {
+  createdAt: Date;
+  chartID: string;
+  isSelected: boolean;
+  onChange: Function;
+}
+
+const Chart: React.FC<ChartProps> = (props: ChartProps) => {
+  const [chartProps, setChartProps] = useState(props);
```

Comment:
is it necessary to store chart props? I assume the only thing you really need isSelected as state?